recipes/wayland/all/test_package/conanfile.py

```diff
@@ -38,6 +38,10 @@ def generate(self):
         pkg_config_deps.generate()
         virtual_build_env = VirtualBuildEnv(self)
         virtual_build_env.generate()
+        if self._has_build_profile:
+            env = Environment()
+            env.append_path("PKG_CONFIG_PATH", self.generators_folder)
```

Comment:
I'll need to understand why this is needed before being comfortable approving - maybe @jcar87 has also some insight, he has been fighting with pkgconfig this last week :)